senders/proxy.go

```diff
@@ -147,6 +148,25 @@ func (sender *proxySender) SendSpan(name string, startMillis, durationMillis int
 	return nil
 }
 
+func (sender *proxySender) SendEvent(name string, startMillis, endMillis int64, source string, tags map[string]string, setters ...event.Option) error {
+	if sender.metricHandler == nil {
```

Comment:
Create a separate eventHandler and use that. Events will be on a different port than the metrics. Should also change the `ProxyConfiguration` to support an `EventsPort`.

Reply:
right now is the same port